pkg/storaged/dialog.jsx

```diff
@@ -465,14 +465,46 @@ export const dialog_open = (def) => {
                 return null;
         })).then(results => {
             const errors = { };
-            fields.forEach((f, i) => { if (results[i]) errors[f.tag] = results[i]; });
+            let scrolled = false;
+            fields.forEach((f, i) => {
+                if (results[i]) {
+                    if (!scrolled) {
+                        show_field(f.tag);
+                        scrolled = true;
+                    }
+                    errors[f.tag] = results[i];
+                }
+            });
             if (Object.keys(errors).length > 0)
                 return Promise.reject(errors);
         });
     };
 
     const dlg = show_modal_dialog(props(), footer_props(null, null));
 
+    function show_field(tag, align) {
+        function scroll() {
+            const dialog_element = document.querySelector('#dialog .pf-v5-c-modal-box__body');
+            const field_element = document.querySelector('#dialog [data-tag="' + tag + '"]');
+
+            if (field_element) {
+                if (dialog_element) {
+                    // check if we need to scroll
+                    const dialog_rect = dialog_element.getBoundingClientRect();
+                    const field_rect = field_element.getBoundingClientRect();
+                    if (field_rect.top >= dialog_rect.top && field_rect.bottom <= dialog_rect.bottom)
+                        return;
+                }
+
+                field_element.scrollIntoView({ behavior: "smooth", block: align || "start" });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20060-20240220-103945-5d97d4e0-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)